deployment/human-connection/service-nginx.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: nitro-web
```

We have a service with that name already
```suggestion
  name: maintenance
```